app/javascript/elm/tests/TimeRangeTests.elm

```diff
@@ -51,14 +51,14 @@ all =
                     |> Expect.equal expected
         , test "viewTimeFilter has a button" <|
             \_ ->
-                TimeRange.view Msg True (Path.fromString "tooltip-icon.svg")
+                TimeRange.view Msg True (Path.fromString "tooltip-icon.svg") (Path.fromString "reset-icon.svg")
                     |> Query.fromHtml
                     |> Query.find [ tag "button" ]
                     |> Event.simulate Event.click
                     |> Event.expect Msg
         , fuzz bool "may be disabled" <|
             \isDisabled ->
-                TimeRange.view (\_ -> ()) isDisabled (Path.fromString "tooltip-icon.svg")
+                TimeRange.view (\_ -> ()) isDisabled (Path.fromString "tooltip-icon.svg") (Path.fromString "reset-icon.svg")
```

Comment:
Since it's not really relevant to this test what kind of icons these are, you could move this to TestUtils.elm and just have one method like
```
defaultIcon = Path.fromString "whatever"
```